### PR TITLE
Add feedback form

### LINKED ISSUE #19: Add "Contact Me" form
Guest User should be able to send message to Application Owner using "Contact Me" form.
Form fields should be:
- name
- email
- message

### PR DESCRIPTION
**Pull request by RasimKhusaenov**
### Summary

- Add "Contact Me" page
- Add FeedbackMailer
- Add Sidekiq
- Replace "byebug" with "binding.pry"

### How it works

image
image
image

### Test plan

List of steps to manually test introduced functionality:

- Go to http://localhost:3000/feedback/new
- Fill form
- Click "Send"
- You are awesome!

### Deploy notes

N/A

### Checklist:

- [x] I have performed a self-review of my own code
- [x] I have added tests for new functionality
- [x] My pull request is shorter than 500 lines

### References
- Resolves #19 Issue

### CODE REVIEW

File: spec/features/feedbacks/create_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  scenario "checking letter body" do
+    expect(current_email).to have_content("Feedback from #{feedback_attributes[:author]}")
+    expect(current_email).to have_content(feedback_attributes[:message])
+  end
+
+  scenario "viewing success sending message" do
+    expect(page).to have_content("Feedback was successfully sent!")
+  end
```
[the remark this answers is not in the document]
Comment (RasimKhusaenov) [reply spec/features/feedbacks/create_spec.rb:33]: Yes, but in that case, I need to change the rule of the rubocop, I suppose. Do you think it's worth it?

[the remark this answers is not in the document]
Comment (RasimKhusaenov) [reply spec/features/feedbacks/create_spec.rb:33]: Okay, I've read [the best practice](https://github.com/rubocop/rspec-style-guide#expectation-per-example) about it. I agree with you.

File: config/routes.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@
   resources :registrations, only: %i[new create]
 
   resources :articles, only: %i[index show]
+  resource :feedback, only: %i[new create]
```
Comment (RamilyaNigmatullina) [inline config/routes.rb:12]: Can we use here?
```suggestion
  resources :feedbacks, only: %i[new create]
```

**Review by RamilyaNigmatullina — APPROVED** [review]

File: config/routes.rb (hunk 2 above)
Comment (RasimKhusaenov) [reply config/routes.rb:12]: Why resources? We can only create one feedback on this page, and it is not saved anywhere.

Comment (RamilyaNigmatullina) [reply config/routes.rb:12]: hmm, ok. we can leave it as is.

**Pull request merged**